developer-experience/recipes/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,60 @@
+
+# Bridge to Kubernetes
+
+Bridge to Kubernetes allows developers to seamlessly work on services both in a shared cluster and locally in debug mode.
+
+To learn more about Bridge to Kubernetes:
+
+- [Introduction](https://code.visualstudio.com/docs/containers/bridge-to-kubernetes)
+
+- [How it works](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes)
+
+- [Configuration](https://docs.microsoft.com/en-us/visualstudio/containers/configure-bridge-to-kubernetes)
+
+- [VS Code Setup](https://docs.microsoft.com/en-us/visualstudio/containers/bridge-to-kubernetes?view=vs-2019)
+
+- [Repository](https://github.com/Microsoft/mindaro)
+
+## Useful Scenarios
+
+We believe this technology can be most helpful...
+
+- if a developer needs to test/debug a new or changed service against an ecosystem of services.
+
+- if a developer does not have the domain knowledge to deploy all the micro services. This allows them to work against a "blackbox" set of already deployed services.
+
+We found the technology very easy to implement so it can probably be used for a wide of cases with minimal training.
+
+## Limitations
+
+While many of these limitations are in the backlog, currently, you cannot use Bridge to Kubernetes in these circumstances...
+
+- if your services need to have replicas.
+
+- if your services require HTTPS and you are running in ISOLATION mode (ex. Istio mTLS).
+
+- if your Pods have more than 1 container (ex. sidecars, Istio).
+
+- if you are using WSL.
+
+- if you are using dev-containers.
+
+## Tested
+
+We have tested the following scenarios with and without ISOLATION mode:
+
+- from local service to shared service.
+
+- from shared service to local service.
+
+- from local service (computer A) to local service (computer B) - routing across AKS cluster.
+
+## Contact
+
+If you have questions, feel free to reach out to:
```

Comment:
nit: we shouldn't add personal information in here - add this contact info to the bridge to kubernetes repo instead.  This is supposed to be read by many people outside of CSE with no connection to CSE - how would they contact a Peter Lasne in CSE? or know who that is

Reply:
Frankly the only reason this document exists is because someone decided it would be valuable as a hack to validate that the bridge-to-kubernetes functionality worked as designed, was easy to use, and determine the revenant use-cases for CSE. Our team performed the hack and wrote the documentation in the code-with-engineering-playbook as discussed.

If someone outside CSE reads this and doesn't know who Peter or Lila is, then they cannot contact us and that is fine. But I was under the impression that generally this was for CSE and if they wanted more info on the hack they could contact us.

Reply:
If that is the case - this content probably fits better in the CWC wiki - This playbook is supposed to be consumable by externals as well - both standalone and as leave behinds for partners we work with. From what you are mentioning here, this sounds like more of a CSE only resource - which fits better to the CWC wiki.

Thoughts @fnocera

Reply:
@plasne I tend to agree with @TessFerrandez that this might be better suited to the CWC wiki vs the playbook.
@charleszipp what are your thoughts on this given you are leading the developer experience portion of the playbook?

Reply:
I agree about removing the contact information. As @TessFerrandez mentioned @plasne this content is intended to be consumable by the public. For that reason, I wouldnt put contact information as it would give the impression to the public that the two of you are supporting the feature. Additionally, I dont think we would want to give a direct email line of communication to the PG. 

I would suggest replacing this with something that directs people to post issues in the official github repo. There, the PG can triage issues as they are opened.